backend/app/usecase/url/urlupdater_test.go

```diff
@@ -0,0 +1,225 @@
+// +build !integration all
+
+package url
+
+import (
+	"testing"
+	"time"
+
+	"github.com/short-d/app/fw/assert"
+	"github.com/short-d/app/fw/timer"
+	"github.com/short-d/short/backend/app/entity"
+	"github.com/short-d/short/backend/app/usecase/keygen"
+	"github.com/short-d/short/backend/app/usecase/repository"
+	"github.com/short-d/short/backend/app/usecase/risk"
+	"github.com/short-d/short/backend/app/usecase/validator"
+)
+
+func TestURLUpdaterPersist_UpdateURL(t *testing.T) {
+	t.Parallel()
+
+	now := time.Now().UTC()
+	alias := "boGp9w35"
+	validNewAlias := "eBJRJJty"
+	validNewLongLink := "https://httpbin.org/get?p1=v1"
+
+	testCases := []struct {
+		name          string
+		alias         *string
```

Comment:
Can this be `nil`? If not, please avoid using pointer and take the address (&) of the url inside the test case.